src/test/theme.test.js

```diff
@@ -1,13 +1,15 @@
 // @flow
+import jsdom from 'mocha-jsdom';
 import React from 'react'
-import { render } from 'enzyme'
+import { mount as render } from 'enzyme'
```

Comment:
Why do we rename this to `render`? That's a separate mode, let's rename all instances of `render` to `mount` instead, otherwise this could be confusing.